pkgs/by-name/di/dillo/package.nix

```diff
@@ -88,6 +80,6 @@ stdenv.mkDerivation (finalAttrs: {
     mainProgram = "dillo";
     maintainers = with lib.maintainers; [ AndersonTorres ];
```

Comment:
Strange...
I am no longer maintaining this.
Please rebase.

Reply:
done